### PR TITLE
fix: add search widgets to TrackerControl overview + fix resetAll button not linked to a function

### PR DESCRIPTION
**Pull request by fquellec**
This issue is related to #563.

- add search widget to app and company name
- fix resetAll button for TrackerControl
- refactor css, add all redundant CSS to Chartview style

### CODE REVIEW

File: experiences/components/chart/ChartView.vue [hunk 1]
```diff
@@ -33,5 +33,43 @@ export default {
...
 
 <style>
+/* Import style for dc graphs */
 @import 'assets/styles/dc.css';
+
+/* Global style for every graph */
+body {
+  font-family: sans-serif;
+  color: #22313f;
+}
```
Comment (valentinoli) [inline experiences/components/chart/ChartView.vue:43]: This block has no effect, can be removed

**Comment by valentinoli** [comment]
image

Inconsistent style

**Comment by valentinoli** [comment]
@fquellec you may look at my changes if you want and then merge if you are OK with them

**Comment by fquellec** [comment]
Nice changes, Thanks !

**Pull request merged**